### PR TITLE
Add fluentbit as a component

### PR DESCRIPTION
**Pull request by philipnrmn**
## High-level description

Adds fluentbit as a component

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

  - [DCOS_OSS-4421](https://jira.mesosphere.com/browse/DCOS_OSS-4421) Add fluentbit to DC/OS as a component

## Related tickets (optional)

Other tickets related to this change:

  - [DCOS-44454](https://jira.mesosphere.com/browse/DCOS-44454)     Add a service account for fluentbit

## Checklist for all PRs

  - [ ] Added a comprehensible changelog entry to `CHANGES.md` or explain why this is not a user-facing change:
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [ ] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [ ] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

File: packages/fluentbit/extra/dcos-fluentbit.service [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Fluentbit: collects and forwards logs
```
Comment (branden) [inline packages/fluentbit/extra/dcos-fluentbit.service]: nit: "Fluent Bit" seems to be the preferred human-readable formatting, judging from docs.

File: packages/fluentbit/extra/start_fluentbit.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+set -o nounset
+
+# Start fluentbit
+exec /opt/mesosphere/bin/fluent-bit "$@"
```
Comment (branden) [inline packages/fluentbit/extra/start_fluentbit.sh]: Are we going to need to put anything in this startup script? I wonder if we can do without it.

**Review by branden — APPROVED** [review]
A couple nits, but overall this LGTM.

File: packages/fluentbit/extra/start_fluentbit.sh (hunk 2 above)
Comment (philipnrmn) [reply packages/fluentbit/extra/start_fluentbit.sh]: I believe you're right.

**Review by gracedo — APPROVED** [review]

**Review by branden — APPROVED** [review]

**Review by branden — APPROVED** [review]

**Pull request merged**